src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -44,10 +45,10 @@
      *            The id of this Backpack
      */
     public PlayerBackpack(@Nonnull PlayerProfile profile, int id) {
-        this(profile, id, profile.getConfig().getInt("backpacks." + id + ".size"));
+        this(profile, id, profile.getConfig().getInt(PlayerBackpack.backpacks + id + ".size"));
```

Comment:
`PlayerBackpack` is redundant here, no need to specify the class you are already in.

Reply:
ff83a0183